src/Dockerfile.dev

```diff
@@ -0,0 +1,16 @@
+FROM microsoft/dotnet:2.1.302-sdk
```

Comment:
Is it common to use docker during development? I personally don't, it seems like an extra hoop. If so, let's stick to one Dockerfile.

Reply:
@Baklap4 multi stage build (one stage on a dev image and the final stage on a runtime image ?

Reply:
@loic-sharma It's very common to use docker during development, as this will make sure if it works on your computer it works on everyone else's

@WernerMairl thanks for the suggestion will look into multistage, this would deffo reduce the build of the `Dockerfile.dev` and shrink it to 60mb (due to the underlying base image dotnet-runtime-xxx